src/resolvers/PublicationResolver.ts

```diff
@@ -6,42 +6,42 @@ import { Social } from '../common/types';
 
 @Resolver((_of) => Publication)
 class PublicationResolver {
-  @Query((_returns) => [Publication], { nullable: false })
+  @Query((_returns) => [Publication], { nullable: false, description: 'Returns a list of all <Publications>' })
   async getAllPublications() {
     return PublicationRepo.getAllPublications();
   }
 
-  @Query((_returns) => Publication, { nullable: true })
+  @Query((_returns) => Publication, { nullable: true, description: 'Returns a single <Publication> via a given <id>' })
   async getPublicationByID(@Arg('id') id: string) {
     return PublicationRepo.getPublicationByID(id);
   }
 
-  @Query((_returns) => Publication, { nullable: true })
+  @Query((_returns) => Publication, { nullable: true, description: 'Returns a single <Publication> via a given <slug>' })
   async getPublicationBySlug(@Arg('slug') slug: string) {
     return PublicationRepo.getPublicationBySlug(slug);
   }
 
-  @Query((_returns) => [Publication])
+  @Query((_returns) => [Publication], {description: 'Returns a list of <Publications> via a given list of <ids>'})
   async getPublicationsByIDs(@Arg('ids', (type) => [String]) ids: string[]) {
     return PublicationRepo.getPublicationsByIDs(ids);
   }
 
-  @FieldResolver((_returns) => Article, { nullable: true })
+  @FieldResolver((_returns) => Article, { nullable: true, description: 'The most recent <Article> of a <Publication>' })
   async mostRecentArticle(@Root() publication: Publication): Promise<Article> {
     return PublicationRepo.getMostRecentArticle(publication);
   }
 
-  @FieldResolver((_returns) => Number)
+  @FieldResolver((_returns) => Number, {description: 'The total shoutouts of a <Publication\'s> <Articles>'})
   async shoutouts(@Root() publication: Publication): Promise<number> {
```

Comment:
numShoutouts to stay consistent with the naming of numArticles

Reply:
frontend is swamped rn, let's not make this change until later because it'll break our API